x-pack/plugins/security_solution/public/entity_analytics/common/utils.ts

```diff
@@ -26,6 +27,27 @@ export const RISK_SEVERITY_COLOUR: { [k in RiskSeverity]: string } = {
   [RiskSeverity.Critical]: SEVERITY_COLOR.critical,
 };
 
+export const useRiskSeverityColors = (): { [k in RiskSeverity]: string } => {
+  const { euiTheme } = useEuiTheme();
+
+  return {
+    [RiskSeverity.Unknown]: euiTheme.colors.vis.euiColorVisNeutral0, // TODO: this is a closest guess based on severity colors, change to grey20 when available
+    // TODO: update these with V9.0.0 severity palette colors when available / keep if the below are  updated with the palette
+    [RiskSeverity.Low]: euiTheme.flags.hasVisColorAdjustment
+      ? '#54B399'
```

Comment:
Suggestion: We can exchange static values for tokens here too:

| level | static value | token |
|------|------------|-----------|
| low | `#54B399` | `euiTheme.colors.vis.euiColorVis0` |
| moderate | `#D6BF57` | `euiTheme.colors.vis.euiColorVis5` |
| high | `#DA8B45` | `euiTheme.colors.vis.euiColorVis7` |
| critical | `#E7664C` | `euiTheme.colors.vis.euiColorVis9` |


Same for code in [use_criticality_level_colors.ts](https://github.com/elastic/kibana/pull/204007/files#diff-e109f518acd31b06cdb340424bfb5b36f9e0da3c856ba72340bb06ee073fdd5c)

Reply:
Nice spot! Thanks will add those in!